grails-app/controllers/com/unifina/controller/api/StreamApiController.groovy

```diff
@@ -168,4 +172,26 @@ class StreamApiController {
 			action.call(stream)
 		}
 	}
+
+	@StreamrApi
+	def status() {
+		Stream s = Stream.get((String) params.id)
```

Comment:
Btw you can have the `id` in the parameter list of the method, e.g., `def status(String id)`. This works for variables defined in the url mapping in `UrlMappings.groovy`. Not saying it needs to be done in that way though, either is fine imo.